### ISSUE TITLE
Publish documentation on docs.asciidoctor.org

### ISSUE THREAD

**Issue by slonopotamus** [asciidoctor/asciidoctor-pdf#1887]
Subj. Creating this issue just for tracking purposes, I'm not likely to work on this in nearest future.

**Comment by mojavelinux** [comment]
:100: 

I do plan to split out the pages in the first round. That provides an opportunity for me to review the content before it gets published to docs.asciidoctor.org. And we're still waiting on Asciidoctor PDF 2, so I'd like it to coincide with that release (since I'm leaning towards not publishing the 1.5.x docs there).

I think the theming guide also needs to be split. The reference for the theme keys would likely be all one page, then the other sections in the document would get their own pages (where it makes sense). There's no doubt in my mind that "theming" should be it's own module.

**Comment by graphitefriction** [comment]
Since I was deep in PDF land writing the kitchen sink test scenarios for 2.0, I decided to start migrating the README content to the Antora structure while waiting for builds (this was before I saw how big the theming guide was :scream_cat:).

This is what I've done:

* Moved the major topics/sections of the README to pages in the `ROOT` module
* Created a symlink in *ROOT/attachments* for *basic-example.adoc -> ../../../../examples/basic-example.adoc*
* Created a symlink in *ROOT/images* for *example-pdf-screenshot.png -> ../../../../examples/example-pdf-screenshot.png*
* Moved the major topics/sections of the theming guide to pages in the `theme` module
  * I didn't create one page for all the keys, because it would be huge, and I, personally don't find the division between key prefixes in the tables particularly easy to differentiate visually and I don't think Algolia searches at that level. Instead, I'm toying with breaking out keys like I did on the Header and Footer Keys and the Title Page Keys.
* Moved the "Extending the Converter" sections in the theming guide to pages in the `extend` module
* Set up antora.yml as follows:

```
name: pdf-converter
title: Asciidoctor PDF
version: '2.0'
prerelease: '.0-alpha.1'
asciidoc:
  attributes:
    xrefstyle: short@
    table-caption: false
    release-line: '2.0.x'
    url-gem: https://rubygems.org/gems/asciidoctor-pdf
    url-project-repo: https://github.com/asciidoctor/asciidoctor-pdf
    url-project-issues: https://github.com/asciidoctor/asciidoctor-pdf/issues
    url-prawn: https://prawnpdf.org
    url-prawn-gmagick: https://github.com/packetmonkey/prawn-gmagick
    url-prawn-svg: https://github.com/mogest/prawn-svg
    url-graphicsmagick: https://www.graphicsmagick.org
nav:
  - modules/ROOT/nav.adoc
  - modules/theme/nav.adoc
  - modules/extend/nav.adoc
 ```

What I haven't done in this PR is remove the content from the README and theming guide. We don't want to remove the info from the README until we publish the docs, and then we'll put a link to the docs in the README. And the theming guide URL will need a redirect, or be depopulated and a link to the theming index page added.

In migrating content that was a monolithic document, content holes and odd transitions appear. However, I've purposely restrained myself from doing any content additions/heavy polishing because we know what happens then (I take months and months :stuck_out_tongue_winking_eye: ). I have made a list of critical content updates and reviews for the 2.0 final release, and then a list of less critical content updates/changes/improvements. I'll post my proposed critical content updates in a separate issue.
